src/common/utils.ts

```diff
@@ -0,0 +1,28 @@
+import { BigNumber } from '@polymeshassociation/polymesh-sdk';
+import { isEntity } from '@polymeshassociation/polymesh-sdk/utils';
+
+/**
+ * @hidden
+ */
+export function toHumanObject(obj: unknown): unknown {
+  if (isEntity(obj)) {
+    return obj.toHuman();
+  }
+
+  if (Array.isArray(obj)) {
+    return obj.map(toHumanObject);
+  }
+
+  if (obj instanceof BigNumber && !obj.isNaN()) {
+    return obj.toString();
+  }
+
+  if (obj && typeof obj === 'object') {
+    return Object.keys(obj).reduce(
+      (r, v) => Object.assign(r, { [v]: toHumanObject(obj[v as keyof typeof obj]) }),
```

Comment:
(nit) Is `Object.assign` needed here? I thought it would be OK to assign directly to `r` here since nothing has a reference to it until this loop returns.

OK to leave `assign` for consistency though, immutable objects are needed for front end change detection.